source/ncengine/graphics2/diligent/pass/PassBackend.cpp

```diff
@@ -230,8 +230,8 @@ void PassBackend::RenderMaterial(Diligent::IDeviceContext& context,
     for (auto [staticPass, skinnedPass, staticBatches, skinnedBatches] : passView)
     {
         // PassManifest verifies static/skinned pass pairs specify the same render targets, so we can just choose from either here.
-        BindRenderTarget(context, swapChain, colorSinkBuffer, depthSinkBuffer, staticPass.colorRTIndex, staticPass.depthRTIndex, m_numSamples > 1);
-        ClearRenderTarget(context, swapChain, colorSinkBuffer, depthSinkBuffer, staticPass.colorRTIndex, staticPass.depthRTIndex, m_numSamples > 1);
+        BindRenderTarget(context, swapChain, colorSinkBuffer, depthSinkBuffer, staticPass.colorRTIndex, staticPass.depthRTIndex, staticPass.isMsaa && m_numSamples > 1);
```

Comment:
Not suggesting a change now, but I'd like to fix this in the near future. Identifying a resource based on the intersection of three variables is beyond what I, as a mere mortal, can reason about.

Reply:
Yar, it be gettin' a bit wild.

Reply:
I'm going to leave it for now but with the intention to revisit when we move to deferred